rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1349,6 +1373,19 @@ public Sha256Hash getBtcBlockchainBlockHashAtDepth(int depth) throws BlockStoreE
         return blockAtDepth.getHeader().getHash();
     }
 
+    public byte[] getBtcBlockchainBlockHeaderByHeight(int height) throws BlockStoreException, IOException {
+        Context.propagate(btcContext);
+        this.ensureBtcBlockStore();
+
+        StoredBlock block = btcBlockStore.getStoredBlockAtMainChainHeight(height);
+
+        if (block == null) {
+            return ByteUtil.EMPTY_BYTE_ARRAY;
+        }
```

Comment:
This is not really necessary since the method `serializeBlockHeader` does the same validation

Reply:
OK done